### DISCUSSION TITLE
New

### DISCUSSION THREAD

**Discussion by FireRose0929** [community/community#73214] [New to GitHub]
How do I make my website

**Comment by Zerotask** [comment]
That depends on your skills and the expected website.
If it's a static website with just HTML, CSS and JavaScript, you can use a Static Site Generator, e. g. https://gohugo.io which is markdown-based.
You can then host your Hugo website with https://pages.github.com

**Comment by DevCode15** [comment]
It depends what you want to build, if you want a landing page for example, you can use HTML, CSS, Javascrpt. Or other front-end technologies.

**Comment by nilushamadhuwanthi123** [comment]
> How do I make my website

If you're building a website from scratch, a good starting point is HTML, CSS, and JavaScript.

HTML handles the structure, CSS handles the design and responsive layout, and JavaScript adds interactivity.

For a simple website, you can start with:

1. Create an `index.html` file
2. Add a `style.css` file for styling
3. Add a `script.js` file for interactions
4. Test the website locally in your browser
5. When it's ready, deploy it using GitHub Pages, Netlify, or Vercel.

If you're just getting started, I would recommend building the first version with plain HTML, CSS, and JavaScript before introducing a framework or static site generator. It makes it easier to understand how the web fundamentals work.
